tests/expr_and_series/is_finite_test.py

```diff
@@ -11,15 +11,12 @@
 
 
 @pytest.mark.filterwarnings("ignore:invalid value encountered in cast")
-def test_is_finite_expr(constructor: Constructor, request: pytest.FixtureRequest) -> None:
-    if ("pyspark" in str(constructor)) or "duckdb" in str(constructor):
-        request.applymarker(pytest.mark.xfail)
-    if "polars" in str(constructor) or "pyarrow_table" in str(constructor):
+def test_is_finite_expr(constructor: Constructor) -> None:
+    if any(x in str(constructor) for x in ("polars", "pyarrow_table", "duckdb")):
         expected = {"a": [False, False, True, None]}
-    elif (
-        "pandas_constructor" in str(constructor)
-        or "dask" in str(constructor)
-        or "modin_constructor" in str(constructor)
+    elif any(
+        x in str(constructor)
+        for x in ("pandas_constructor", "dask", "modin_constructor", "pyspark")
```

Comment:
does pyspark end up here because we go via pandas in the constructor?

Reply:
Most likely yes!